### DISCUSSION TITLE
Capacitor 6 ?

### DISCUSSION THREAD

**Discussion by LgArcida** [capacitor-community/bluetooth-le#650] [General]
It's been more than 2 months since this [PR](https://github.com/capacitor-community/bluetooth-le/pull/620) was created.
Will plugin be updated to cap 6 ?

Not sure if `npm i @capacitor-community/bluetooth-le@next` is stable/secure enough

**Comment by pwespi** [comment]
version 6.0.0 of the plugin has been released
